OpenRA.Game/Traits/World/MusicPlaylist.cs

```diff
@@ -45,8 +45,11 @@ public class MusicPlaylist : INotifyActorDisposing, IGameOver
 		readonly MusicInfo[] random;
 		readonly MusicInfo[] playlist;
 
+		public readonly bool IsMusicInstalled;
 		public readonly bool IsMusicAvailable;
 
+		public bool AmbientMusic;
```

Comment:
I would define this as `public bool CurrentSongIsAmbient { get; private set; }` to make its purpose clearer.

This is going a step further than we needed for this pr, but since it's here we might as well set it up in preparation for the next changes.